src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    /// <summary>
+    /// TODO: Fill this out
+    /// </summary>
+    public class DelayedDeliverySettings : ExposeSettings
+    {
+        internal DelayedDeliverySettings(SettingsHolder settings) : base(settings) { }
+
+        /// <summary>
+        /// TODO: Fill this out
+        /// </summary>
+        public void DisableTimeoutManager()
```

Comment:
Do we want to make this a FluentAPI? I assume that both settings can be set at the same time. So in an endpoint can have `DisableTimeoutManager` and `AllEndpointsSupportNativeDelays` at the same time. But our current API forces us to use:
```
var transport = endpointConfiguration.UseTransport<RabbitMQTransport>().ConnectionString("...");
transport.DelayedDelivery().DisableTimeoutManager();
transport.DelayedDelivery().AllEndpointsSupportNativeDelays();
```
It is not as smooth as it could be. We could use fluent API to make it a one liner as:
```
endpointConfiguration.UseTransport<RabbitMQTransport>().ConnectionString("...").DelayedDelivery().DisableTimeoutManager().AllEndpointsSupportNativeDelays();
```

I am not a great fan of FluentAPI but I want to simplify setting both fields for one endpoint. Fluent API is used in other transports like:
https://github.com/Particular/NServiceBus.AzureServiceBus/blob/ccf2f9ed96b433716e84c00972c16fa393968c60/src/Transport/Config/ExtensionPoints/AzureServiceBusMessagingFactoriesSettings.cs
or
https://github.com/Particular/NServiceBus.AzureServiceBus/blob/ccf2f9ed96b433716e84c00972c16fa393968c60/src/Transport/Config/ExtensionPoints/AzureServiceBusMessageReceiverSettings.cs

@bording @SeanFeldman what do you think?